LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2024 Tinylibs
+Copyright (c) 2018 Made With MOXY Lda <hello@moxy.studio>
```

Comment:
Whats that?

Reply:
the license of cross-spawn. any usage of the original code needs to maintain the copyright